vnext/Microsoft.ReactNative/Fabric/Composition/ContentIslandComponentView.cpp

```diff
@@ -47,6 +50,23 @@ void ContentIslandComponentView::OnMounted() noexcept {
       winrt::Microsoft::ReactNative::Composition::Experimental::CompositionContextHelper::InnerVisual(Visual())
           .as<winrt::Microsoft::UI::Composition::ContainerVisual>());
   m_childSiteLink.ActualSize({m_layoutMetrics.frame.size.width, m_layoutMetrics.frame.size.height});
+
+  m_navigationHost = winrt::Microsoft::UI::Input::InputFocusNavigationHost::GetForSiteLink(m_childSiteLink);
+
+  m_departFocusRequestedRevoker =
+      m_navigationHost.DepartFocusRequested(winrt::auto_revoke, [wkThis = get_weak()](const auto &, const auto &args) {
+        if (auto strongThis = wkThis.get()) {
+          const bool next = (args.Request().Reason() == winrt::Microsoft::UI::Input::FocusNavigationReason::First);
```

Comment:
I'd invert this to be `next = reason != Last`.  That way restore and programmatic (and all the directions) will move to the first element, not the last.

Ideally we'd handle hint rects and direction nav etc.  But we're not there yet.